<launch>

    <!-- ========================================================== -->
    <!-- Launch the camera.                                         -->
    <!-- ========================================================== -->

    <param name="camera/image_raw"       type="string" value="jpeg" />

    <node name="camera"        pkg="camera_aravis"  type="camnode" />
<!--
-->
  
</launch>
